### PR TITLE
sprint length change

### PR DESCRIPTION
**Pull request by temaniarpit27**

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -589,7 +603,7 @@ func (c *BorConfig) calculateBorConfigHelper(field map[string]uint64, number uin
 	for i := 0; i < len(keys)-1; i++ {
 		valUint, _ := strconv.ParseUint(keys[i], 10, 64)
 		valUintNext, _ := strconv.ParseUint(keys[i+1], 10, 64)
-		if number > valUint && number < valUintNext {
+		if number >= valUint && number < valUintNext {
```
Comment (temaniarpit27) [inline params/config.go:606]: need to check if this is a consensus level change or not

File: tests/bor/bor_reorg_test.go [hunk 2]
```diff
@@ -1,278 +0,0 @@
...
-
-var (
-	// addr1 = 0x71562b71999873DB5b286dF957af199Ec94617F7
-	pkey1, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
-	// addr2 = 0x9fB29AAc15b9A4B7F17c3385939b007540f4d791
-	pkey2, _ = crypto.HexToECDSA("9b28f36fbd67381120752d6172ecdcf10e06ab2d9a1367aac00cdcd6ac7855d3")
-	keys     = []*ecdsa.PrivateKey{pkey1, pkey2}
-)
-
-func initMiner(genesis *core.Genesis, privKey *ecdsa.PrivateKey) (*node.Node, *eth.Ethereum, error) {
```
Comment (JekaMas) [inline tests/bor/bor_reorg_test.go:39]: Why this was removed?

**Review by JekaMas — APPROVED** [review]

**Comment by temaniarpit27** [comment]
@0xKrishna any update on the self heal issue on heimdall?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -104,7 +104,7 @@ jobs:
           uses: actions/checkout@v3
           with:
             repository: maticnetwork/matic-cli
-            ref: v0.3.0-dev
+            ref: arpit/pos-655-2
```
Comment (JekaMas) [inline .github/workflows/ci.yml:107]: do we need it like that branch?

**Review by JekaMas — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (temaniarpit27) [reply .github/workflows/ci.yml:107]: yes for now
we will be making changes again in upcoming releases

**Review by marcello33 — APPROVED** [review]

**Pull request merged**